modules/exploits/linux/local/su_login.rb

```diff
@@ -0,0 +1,97 @@
+##
+# This module requires Metasploit: https://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+class MetasploitModule < Msf::Exploit::Local
+    include Msf::Post::Linux
```

Comment:
Please, can you run `rubocop` against your ruby file? It looks like there is an indentation problem:
```
rubocop -a modules/exploits/linux/local/su_login.rb
```

Reply:
Fixed. See commit 38fb6441696662d9aea17d95c8bc30021e002088